src/Illuminate/Auth/TokenGuard.php

```diff
@@ -18,11 +18,11 @@ class TokenGuard implements Guard
     protected $request;
 
     /**
-     * The name of the field on the request containing the API token.
+     * The name of the query string item from the request containing the API token.
      *
      * @var string
      */
-    protected $inputKey;
+    protected $queryKey;
```

Comment:
Doesn't this could potentially be breaking BC, if one extending `Illuminate\Auth\TokenGuard` and changing the `$inputKey`.